api/controllers/PostController.js

```diff
@@ -348,6 +348,34 @@ module.exports = {
       )
     }))
     .then(() => res.ok({}), res.serverError)
+  },
+
+  respond: function (req, res) {
+    var userId = req.session.userId
+    var post = res.locals.post
+    var response = req.param('response')
+
+    EventResponse.query(qb => {
+      qb.where({user_id: userId, post_id: post.id})
+      qb.orderBy('created_at', 'desc')
+      return qb
+    }).fetchAll()
+    .then(eventResponses => {
+      if (eventResponses.length > 0) {
+        var lastResponse = eventResponses.models[0].get('response')
+        return Promise.map(eventResponses.models, eventResponse => { eventResponse.destroy() })
```

Comment:
no need for curly braces around eventResponse.destroy() -- in fact, since it's a one-liner, i would usually name the argument something briefer like `r` or `resp`